servers.json

```diff
@@ -176,6 +176,7 @@
         "name": "ColdNetwork",
         "addresses": [
             "coldnetwork.net"
```

Comment:
You must add a comma after this address entry.

Reply:
hopefully fixed